.github/ISSUE_TEMPLATE/dev_card_template.md

```diff
@@ -6,33 +6,34 @@ labels: ''
 assignees: ''
 
 ---
-
 **Summary:** Development of the XXXXX component.
-Develop on component branch, push to staging and push to prod when approved.
+Develop on component branch, push to staging and flag ready to merge when reviwed.
 
-**Acceptance Criteria:**
+Owner : dev assigned on the card 
 
-- [ ] Component kickoff
-- [ ] Sync with Design
-- [ ] Requirements documentation
-- [ ] Requirements doc: Consider edge cases
-- [ ] Requirements doc: Describe component properties
-- [ ] Review of requirements doc
+**Definition of done:**  
+- [ ] Prior to doing work, clarify scope of ticket with the designers (dev design sync or slack) and adjust if necessary
+- [ ] Check with DTO if there are upcoming changes to this component or potential impact if we were to update the component
+- [ ] Consider edge cases
+- [ ] Describe component properties
+- [ ] Update Requirements doc with agreed specifications
 - [ ] Develop the component according to the requirements doc and design specs
 - [ ] Add component to storybook
 - [ ] Write draft of dev guidance
+- [ ] Handoff from draft dev guidance to documentation lead and tag the content owner 
+- [ ] Code review
+- [ ] After PR approval, merge component to staging
+
+**Acceptance criteria:** 
 - [ ] Sync dev properties with Figma properties (design + dev)
```

Comment:
```suggestion
- [ ] Implement properties as stated in requirements doc
```
Suggestion to change this, the sync of properties should happen in the reqs doc not in the dev ticket